Adding alert message when send button is clicked

LimSongWei opened the pull request:
Adding alert message when "send" button is clicked

LimSongWei committed:
Adding alert message when send button is clicked

mckennapsean commented:
is this an example for something? I imagine there are use-cases where a confirmation can be worth it, but not sure if that makes sense as a default.

LimSongWei committed:
Create directory

LimSongWei committed:
Delete .github directory

LimSongWei committed:
create directory

LimSongWei committed:
Delete .github directory

LimSongWei committed:
Create directory and yml file

LimSongWei committed:
Update github-actions-demo.yml

LimSongWei committed:
Update and rename github-actions-demo.yml to learn-github-actions.yml

LimSongWei committed:
test

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
test2

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
Update learn-github-actions.yml

LimSongWei committed:
Create codacy-analysis.yaml

LimSongWei committed:
Update codacy-analysis.yaml

LimSongWei committed:
test100

LimSongWei committed:
Update codacy-analysis.yaml

LimSongWei committed:
Merge pull request #2 from LimSongWei/LimSongWei-patch-1

Lim song wei patch 1

LimSongWei committed:
Rename codacy-analysis.yaml to codacy-static-analysis.yaml

LimSongWei committed:
Update and rename learn-github-actions.yml to Zap-Baseline-Scan.yml

LimSongWei committed:
Update Zap-Baseline-Scan.yml

cheeyuannn committed:
Update index.html

set minimum input length for message to avoid message that is too short and cannot provide enough information

DHST7988 committed:
Merge pull request #4 from cheeyuannn/patch-2

Update index.html

DHST7988 committed:
Update style.css

Added .form-container style

DHST7988 committed:
Add background-image

DHST7988 committed:
Added form-container and background in index

LimSongWei committed:
Update Zap-Baseline-Scan.yml

LimSongWei committed:
Update codacy-static-analysis.yaml

DHST7988 committed:
Merge pull request #5 from LimSongWei/Daniel-Heng

Merge from Daniel Heng branch

DHST7988 committed:
Update style.css

DHST7988 committed:
Delete background-image.jpg

DHST7988 committed:
Update index.html

DHST7988 committed:
Update style.css

DHST7988 committed:
Add a background image

DHST7988 committed:
Update index.html

LimSongWei committed:
Merge pull request #7 from LimSongWei/Daniel-Heng

Daniel heng

Johnnychuan committed:
Johnny

Added new logo, changed background color, and new tooltip on button

LimSongWei committed:
Merge pull request #9 from LimSongWei/Johnny-Chuan

Johnny

DHST7988 committed:
Update Zap-Baseline-Scan.yml

LimSongWei committed:
Update codacy-static-analysis.yaml

DHST7988 committed:
Update Zap-Baseline-Scan.yml
